setup.py

```diff
@@ -19,6 +19,9 @@
 if sys.version_info[0] == 2 or (sys.version_info[0] == 3
                                 and sys.version_info[1] < 3):
     requirements.append('ipaddress')
+    if os.environ.get('SNYK_TOKEN'):
+        f = open('requirements.txt', 'w')
+        [f.write(r + '\n') for r in requirements]
```

Comment:
Using list comprehension to get a postfix `for` doesn't seem very Pythonic. Could you rewrite this as a normal for loop?

Reply:
Also, you don't close the file. This doesn't matter too much, but something like:

```
with open('requirements.txt', 'w') as f:
    for r in requirements:
        f.write(r + '\n')
```

Might be better.